library/VTKExtensions/Rendering/vtkF3DRenderer.cxx

```diff
@@ -498,7 +498,10 @@ void vtkF3DRenderer::ConfigureGridUsingCurrentActors()
       {
         for (int i = 0; i < 3; i++)
         {
-          double size = bounds[2 * i + 1] - bounds[2 * i];
+          // a small margin is added to the size to avoid z-fighting if large translucent
+          // triangles are exactly aligned with the grid bounds
+          constexpr double margin = 1.0001;
+          double size = margin * (bounds[2 * i + 1] - bounds[2 * i]);
```

Comment:
what does this do ? please mention it in the PR description

Reply:
There's a comment in the code

Reply:
> please mention it in the PR description

and make the life of the person creating the release easier.

Alternatively update the changelog, but that ship has sailed ^^